tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -119,7 +119,7 @@
                     </Grid.ColumnDefinitions>
 
                     <!-- Configuration unit results -->
-                    <Grid RowSpacing="20" BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}" BorderThickness="{ThemeResource TopNavigationViewContentGridBorderThickness}" >
+                    <Grid RowSpacing="20" BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}" BorderThickness="0, 1, 0, 2" >

```

Comment:
Nit: you use this thickness three times in this file, consider making it a resource.